packages/strapi-design-system/webpack.config.js

```diff
@@ -52,6 +53,14 @@ module.exports = {
     library: 'strapiDs',
     umdNamedDefine: true,
   },
+  optimization: {
+    splitChunks: {
+      chunks: 'all',
+      maxInitialRequests: 30,
+      maxAsyncRequests: 30,
+      minSize: 20,
```

Comment:
Curious to know why 20bytes to generate a chunk?

Reply:
If you don't put that webpack won't consider the shared packages as being "big enough" to put them in a common chunk we can play we this to force chunking. 

I'll move it outside of this pr and open a draft one with this